ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-routing.j2

```diff
@@ -7,13 +7,26 @@
 
 | VRF | Routing Enabled |
 | --- | --------------- |
-| default | {{ ip_routing | arista.avd.default(false) }} |
-{%     for vrf in vrfs | arista.avd.natural_sort('name') %}
-{%         if vrf.ip_routing is arista.avd.defined(true) %}
-| {{ vrf.name }} | true |
-{%         elif vrf.ip_routing is arista.avd.defined(false) %}
-| {{ vrf.name }} | false |
+{%     set default_ip_routing = ip_routing | arista.avd.default(false) %}
+{%     set default_ip_routing_ipv6_interfaces = ip_routing_ipv6_interfaces | arista.avd.default(false) %}
+{# Handle the case where VRF default exists #}
+{%     set vrf_default = vrfs | arista.avd.default([]) | selectattr("name", "arista.avd.defined", "default") | first %}
```

Comment:
All this logic is handled in the documentation template, so you can set ip routing for default vrf either with the global keys or with `vrf:default`. Unfortunately that is not how it is implemented in the configuration. So we will not _configure_ ip routing for vrf default, if it is only set under the vrfs.

Reply:
so the point of this comment is to implement this in the eos template as well right?